testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -754,6 +755,22 @@ public void testShowCreateInformationSchemaTable()
                         ")");
     }
 
+    @Test
+    public void testRenameSchema()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_SCHEMA) || !hasBehavior(SUPPORTS_CREATE_SCHEMA)) {
+            throw new SkipException("create/rename schema is not supported by the connector");
+        }
+
+        String schemaName = "test_schema_" + randomTableSuffix();
```

Comment:
nit: It's good to name objects after the test name. e.g. `test_rename_schema_`

Reply:
done